c++/src/kj/iovmax.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2013-2014 Sandstorm Development Group, Inc. and contributors
```

Comment:
Perhaps instead of a new file, we could put this in `miniposix.h`, which is already a cross-platform compatibility file?